dist/Tie-File/Makefile.PL

```diff
@@ -0,0 +1,33 @@
+## This -*- perl -*- script writes the Makefile for Tie::File
+# You should read "perldoc perlmodinstall" for instructions on
+#  how to install modules like this.
+
+require 5.004;
```

Comment:
This should be 5.006, the module itself uses `our`.

Reply:
Fixed. Thanks.